code/modules/mob/living/simple_animal/hostile/carp.dm

```diff
@@ -51,7 +51,7 @@
 	del_on_death = 1
 
 /mob/living/simple_animal/hostile/carp/megacarp
-	icon = 'icons/mob/64x32.dmi'
+	icon = 'icons/mob/64x32mobs.dmi'

```

Comment:
Basically unfixable find me a descriptor for 'wide but not tall' mobs and you will be considered.